packages/config/config/devices/0x010f/fgd212_3.5.json

```diff
@@ -167,315 +173,383 @@
+					"value": 0
+				}
+			]
 		},
 		"19": {
-			"label": "Forced switch on brightness level",
-			"description": "Dimmer 2 will always set this brightness level.",
+			"label": "Forced Switch On Brightness Level",
+			"description": "Percentage value.",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 99,
-			"defaultValue": 0
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "Disable, Previous Value is Used",
+					"value": 0
+				}
+			]
 		},
 		"20": {
-			"label": "Inputs Button/Switch configuration",
-			"description": "Binary inputs type configuration",
+			"label": "Input Button/Switch Configuration",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 255,
+			"maxValue": 2,
 			"defaultValue": 0,
 			"unsigned": true,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Mono-stable input (button)",
+					"label": "Mono-stable Input (Button)",
 					"value": 0
 				},
 				{
-					"label": "Bi-stable input (switch)",
+					"label": "Bi-stable Input (Switch)",
 					"value": 1
 				},
 				{
-					"label": "Roller blind switch (UP / DOWN)",
+					"label": "Roller Blind Switch (UP / DOWN)",
 					"value": 2
 				}
 			]
 		},
 		"21": {
-			"label": "Value sent to associated devices on single click",
+			"label": "Value Sent to Associated Devices on Single Click",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 255,
+			"maxValue": 1,
 			"defaultValue": 0,
 			"unsigned": true,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Set associated devices to last state",
+					"label": "Set Associated Devices to Last State",
 					"value": 0
 				},
 				{
-					"label": "Dimmer 2 state synchronizes brightness level",
+					"label": "This Dimmers Current State Synchronizes Brightness Level",
 					"value": 1
 				}
 			]
 		},
 		"22": {
-			"label": "Change [On-Off] bi-stable keys",
+			"label": "Change Bi-stable Input Switch Behavior",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 255,
+			"maxValue": 1,
 			"defaultValue": 0,
 			"unsigned": true,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Device changes status on key status change",
+					"label": "Output Changes on Input Switch Change",
 					"value": 0
 				},
 				{
-					"label": "Status depends on key status",
+					"label": "Output Synced with Input Switch Position",
 					"value": 1
 				}
 			]
 		},
 		"23": {
-			"label": "Double click option",
-			"description": "Double-click set lighting at 100%",
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Double Click Sets Brightness to 100%",
+			"defaultValue": 1
+		},
+		"24[0x01]": {
+			"label": "Send Command Frames in 2nd and 3rd Association Groups when Switching ON (Single Click)",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 255,
-			"defaultValue": 1,
+			"maxValue": 1,
+			"defaultValue": 0,
 			"unsigned": true,
 			"allowManualEntry": false,
 			"options": [
```

Comment:
I think you should be able to base these partials off
`"$import": "../templates/master_template.json#base_enable_disable"`
so you'd just have to override the label and default value where necessary.